### ISSUE TITLE
Queries regarding train_brisque.py code

### ISSUE THREAD

**Issue by spaul13** [buyizhiyou/NRVQA#4]
First of all thanks a lot for adding the BRISQUE training code. Can u please tell me how to generate info.txt as I want to train BRISQUE on a custom dataset?

can u please tell me which part of the train_brisque.py code is extracting NSS (Natural Scene statistics) as what I know we have to extract NSS before training the BRISQUE model.

Any help will be highly appreciated.

**Comment by buyizhiyou** [comment]
> First of all thanks a lot for adding the BRISQUE training code. Can u please tell me how to generate info.txt as I want to train BRISQUE on a custom dataset?
> 
> can u please tell me which part of the train_brisque.py code is extracting NSS (Natural Scene statistics) as what I know we have to extract NSS before training the BRISQUE model.
> 
> Any help will be highly appreciated.

Hi!
You can download LIVE dataset which contains info.txt.Also you can label your own dataset similar to LIVE dataset format.
In this project,brisque.py aims to extract NSS.

**Comment by spaul13** [comment]
Thanks a lot @buyizhiyou  for the reply. Can u please tell me how to run the SVR training on GPU?

**Comment by buyizhiyou** [comment]
> Thanks a lot @buyizhiyou for the reply. Can u please tell me how to run the SVR training on GPU?

Maybe  you can use tensorflow or pytorch to build a SVR model and train your datasets on GPU!
